examples/nodejs/doc-examples-js-apis.ts

```diff
@@ -174,6 +178,45 @@ async function example_API_Delete(cacheClient: CacheClient) {
   }
 }
 
+async function example_API_Increment(cacheClient: CacheClient) {
+  const result = await cacheClient.increment('test-cache', 'test-key', 1);
```

Comment:
might be worh doing a `set` before this, to some non-zero value, and then log the new value in your log statement?  It should be available as part of the Success result.